docs/logging-usage.rst

```diff
@@ -250,10 +250,11 @@ Create a Cloud Pub/Sub sink:
 
    >>> from gcloud import logging
    >>> client = logging.Client()
+
    >>> sink = client.sink(
    ...     "robots-pubsub",
-   ...     filter='log:apache-access AND textPayload:robot')
-   >>> sink.pubsub_topic = 'projects/my-project/topics/my-topic'
+   ...      'log:apache-access AND textPayload:robot', 
```

Comment:
deleted space at end, I'm not sure what specifically you mean by it being different. I just manually entered spaces to align it with the lines above.